scripts/generator/graphql_generator/csharp/SDK/iOS/iOSNativePayable.cs.erb

```diff
@@ -0,0 +1,66 @@
+#if UNITY_IOS
+namespace <%= namespace %>.SDK.iOS {
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using System.Runtime.InteropServices;
+    using <%= namespace %>.SDK;
+
+    class iOSNativePayable : INativePayable {
+        [DllImport ("__Internal")]
+        private static extern bool _CanCheckoutWithApplePay();
+
+        [DllImport ("__Internal")]
+        private static extern bool _CanShowApplePaySetup();
+
+        [DllImport ("__Internal")]
+        private static extern void _ShowApplePaySetup();
+
+        [DllImport ("__Internal")]
+        private static extern bool _CreateApplePaySession(string merchantID, string countryCode, string currencyCode, string unityDelegateObjectName, string serializedSummaryItems, string serializedShippingMethods, bool requiringShipping);
+
+        [DllImport ("__Internal")]
+        private static extern void _PresentApplePayAuthorization();
+
+        private Cart CurrentCart;
+
+        public iOSNativePayable(Cart cart) {
```

Comment:
I'd like to see this filled out a bit more before we make up our minds - but still wondering if the way the cart get information is via events dispatched from this class (and then we can avoid having to pass the cart here.)